tests/IssueTest.php

```diff
@@ -0,0 +1,175 @@
+        $issue->comment_count = 5;
+        $issue->completed = false;
+
+        $expected = json_encode([
+            'id' => 7,
+            'title' => "A Title",
+            'summary' => "A summary of an issue",
+            'severity' => "Severe",
+            'date_created' => "05/11/2024",
+            'comment_count' => 5,
+            'completed' => false,
+        ]);
+
+        $actual = json_encode($issue);
+
+        $this->assertEquals($expected, $actual);
+    }
+
+    public function testIssueJsonSerialize_longDescriptionSuccess()
+    {
+        $issue = new Issue();
+
+        $issue->id = 7;
+        $issue->title = "A Title";
+        $issue->summary = "In a world where technology evolves rapidly, it's essential to keep learning and adapting to new ski";
+        $issue->severity = "Severe";
+        $issue->date_created = "2024-11-05 15:56:55";
+        $issue->comment_count = 5;
+        $issue->completed = false;
+
+        $expected = json_encode([
+            'id' => 7,
+            'title' => "A Title",
+            'summary' => "In a world where technology evolves rapidly, it's essential to keep learning and adapting to new ski",
+            'severity' => "Severe",
+            'date_created' => "05/11/2024",
+            'comment_count' => 5,
+            'completed' => false,
+        ]);
+
+        $actual = json_encode($issue);
+
+        $this->assertEquals($expected, $actual);
+    }
+
+    public function testIssueJsonSerialize_nullableDescriptionSuccess()
+    {
+        $issue = new Issue();
+
+        $issue->id = 7;
+        $issue->title = "A Title";
+        $issue->summary = null;
+        $issue->severity = "Severe";
+        $issue->date_created = "2024-11-05 15:56:55";
+        $issue->comment_count = 5;
+        $issue->completed = false;
+
+        $result = $issue->jsonSerialize();
+
+        $this->assertNull($result['summary']);
+    }
+
+    public function testIssueJsonSerialize_nullableCommentCountSuccess()
+    {
+        $issue = new Issue();
+
+        $issue->id = 7;
+        $issue->title = "A Title";
+        $issue->summary = "Hello";
+        $issue->severity = "Severe";
+        $issue->date_created = "2024-11-05 15:56:55";
+        $issue->comment_count = null;
+        $issue->completed = false;
+
+        $result = $issue->jsonSerialize();
+
+        $this->assertNull($result['comment_count']);
+    }
+
+    public function testIssueJsonSerialize_completedTrueSuccess()
+    {
+        $issue = new Issue();
+
+        $issue->id = 7;
+        $issue->title = "A Title";
+        $issue->summary = null;
+        $issue->severity = "Severe";
+        $issue->date_created = "2024-11-05 15:56:55";
+        $issue->comment_count = 5;
+        $issue->completed = true;
+
+        $result = $issue->jsonSerialize();
+
+        $this->assertTrue($result['completed']);
+    }
+
+    public function testIssueJsonSerialize_completedFalseSuccess()
```

Comment:
Your test `testIssueJsonSerialize_success` gives you coverage of the `false` path.

Reply:
done